tools/ibc-test/src/framework/utils/common.rs

```diff
@@ -62,3 +64,17 @@ pub fn transfer_port_id(chain_type: ChainType) -> PortId {
         }
     }
 }
+
+pub fn prepare_cell_emitter(axon_port: u16, ckb_port: u16) -> Result<(), Error> {
```

Comment:
[2023-12-19T11:48:10Z WARN  emitter::global_state] Failed to open state db, file: "/tmp/emitter/scan_state", error: Os { code: 2, kind: NotFound, message: "No such file or directory" }

see https://github.com/synapseweb3/forcerelay/actions/runs/7261303916/job/19782206216#step:19:134

Reply:
handling on this